docs/source/app_developers_guide/docker.rst

```diff
@@ -0,0 +1,417 @@
+  $ intkey load -f batches.intkey -U http://rest_api:8080
+
+You can observe the processing of the intkey transactions by observing the
+logging output of the intkey transaction processor and the validator, from the
+terminal window in which you ran the *docker-compose* command. A truncated
+example of the output:
+
+.. code-block:: console
+
+  tp_intkey_python_1  | [21:02:53.164 DEBUG    handler] Incrementing "VaUEPt" by 1
+  validator_1         | [21:02:53.169 DEBUG    interconnect] ServerThread receiving TP_STATE_SET_REQUEST message: 194 bytes
+  validator_1         | [21:02:53.171 DEBUG    tp_state_handlers] SET: ['1cf126d8a50604ea6ab1b82b33705fc3eeb7199f09ff2ccbc52016bbf33ade68dc23f5']
+  validator_1         | [21:02:53.172 DEBUG    interconnect] ServerThread sending TP_STATE_SET_RESPONSE to b'63cf2e2566714070'
+  validator_1         | [21:02:53.176 DEBUG    interconnect] ServerThread receiving TP_PROCESS_RESPONSE message: 69 bytes
+  validator_1         | [21:02:53.177 DEBUG    interconnect] message round trip: TP_PROCESS_RESPONSE 0.042026519775390625
+  validator_1         | [21:02:53.182 DEBUG    interconnect] ServerThread sending TP_PROCESS_REQUEST to b'63cf2e2566714070'
+  tp_intkey_python_1  | [21:02:53.185 DEBUG    core] received message of type: TP_PROCESS_REQUEST
+  validator_1         | [21:02:53.191 DEBUG    interconnect] ServerThread receiving TP_STATE_GET_REQUEST message: 177 bytes
+  validator_1         | [21:02:53.195 DEBUG    tp_state_handlers] GET: [('1cf126721fff0dc4ccb345fb145eb9e30cb7b046a7dd7b51bf7393998eb58d40df5f9a', b'\xa1fZeuYwh\x1a\x00\x019%')]
+  validator_1         | [21:02:53.200 DEBUG    interconnect] ServerThread sending TP_STATE_GET_RESPONSE to b'63cf2e2566714070'
+  tp_intkey_python_1  | [21:02:53.202 DEBUG    handler] Incrementing "ZeuYwh" by 1
+
+
+Config Transaction Family Usage
+===============================
+
+Sawtooth provides a :doc:`config transaction family
+<../transaction_family_specifications/config_transaction_family>` that stores on-
+chain configuration settings, along with a config family transaction
+processor written in Python.
+
+One of the on-chain settings is the list of supported transaction families.
+In the example below, a JSON array is submitted to the `sawtooth config`
+command, which creates and submits a batch of transactions containing the
+configuration change.
+
+The JSON array used tells the validator or validator network to accept transactions of the following types:
+
+* intkey
+* sawtooth_config
+
+To create and submit the batch containing the new configuration, enter the
+following commands: 
+
+.. note::
+
+  The config command needs to use a key generated in the validator container.
+  Thus, you must open a terminal window running in the validator container,
+  rather than the client container (for the following command only).
+  Run the following command from your host machine's CLI:
+
+.. code-block:: console
+
+  % docker exec -it compose_validator_1 bash
```

Comment:
Same as my note above, we should merge #572 and then update the container names.